Userland/Libraries/LibWebView/SearchEngine.cpp

```diff
@@ -17,6 +17,7 @@ static constexpr auto builtin_search_engines = Array {
     SearchEngine { "Ecosia"sv, "https://ecosia.org/search?q={}"sv },
     SearchEngine { "GitHub"sv, "https://github.com/search?q={}"sv },
     SearchEngine { "Google"sv, "https://www.google.com/search?q={}"sv },
+    SearchEngine { "Kagi"sv, "https://kagi.com/search?q="sv },
```

Comment:
Should it be this instead?
```suggestion
    SearchEngine { "Kagi"sv, "https://kagi.com/search?q={}"sv },
```

Reply:
Yes it should. Please amend the commit @mrdcbrush. Otherwise this looks good to me.